Makefile

```diff
@@ -1,7 +1,7 @@
 # Image URL to use all building/pushing image targets
 IMG ?= kubeflow/training-operator:latest
 # CRD generation options
-CRD_OPTIONS ?= "crd:generateEmbeddedObjectMeta=true"
+CRD_OPTIONS ?= "crd:generateEmbeddedObjectMeta=true,maxDescLen=300"
```

Comment:
@johnugeorge I tried to set 300 characters. How about generated CRD descriptions?

Reply:
I see that there are some minor unintended line breaks.   https://github.com/kubeflow/training-operator/pull/1735/files#diff-f68a63b0953eb712985f398c6c8ae96093830f251ec4ced3865cbe46d41550b3L934  This doesn't seem to be a major concern. Good to go

Reply:
I increased it to 400 characters. The unintended line breaks that you pointed out are resolved.